### PR TITLE
DOCS-1428: CE-Support-3.17.0

### PR DESCRIPTION
**Pull request by bmckercher123**
Replaces this PR: https://github.com/tigera/docs/pull/669

Product Version(s):

Issue:

Link to docs preview:
(https://deploy-preview-704--tigera.netlify.app/calico-enterprise/next/getting-started/compatibility)

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 1]
```diff
@@ -37,79 +37,46 @@ Note that all Windows feature limitations are described in [Windows limitations]
...
-| 3.10                 | Azure CNI with {{prodname}} network policy                                               |
-| 3.9                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.8                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.7                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.6                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.5                  | Azure CNI with {{prodname}} network policy                                               |
+| {{prodname}} version | {{prodname}} support                                         |
+| -------------------- | ------------------------------------------------------------ |
+| 3.14 through current release         | - {{prodname}} CNI with network policy<br />- Azure CNI with {{prodname}} network policy |
+| 3.25 to 3.13         | Azure CNI with {{prodname}} network policy                   |
```
Comment (girishshankaran) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx]: Just wondering if this should be 3.13 to 3.25.

Also, I do not see the version 3.17 in the deploy preview.

**Review by girishshankaran — APPROVED** [review]
LGTM

File: calico-enterprise/getting-started/compatibility.mdx [hunk 2]
```diff
@@ -37,79 +37,46 @@ Note that all Windows feature limitations are described in [Windows limitations]
...
-| 3.10                 | Azure CNI with {{prodname}} network policy                                               |
-| 3.9                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.8                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.7                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.6                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.5                  | Azure CNI with {{prodname}} network policy                                               |
+| {{prodname}} version | {{prodname}} support                                         |
+| ---------------------------- | ------------------------------------------------------------ |
+| 3.14 through current release | - {{prodname}} CNI with network policy<br />- Azure CNI with {{prodname}} network policy |
+| 3.13 to 3.25                 | Azure CNI with {{prodname}} network policy                   |
```
Comment (sujeet-kr) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
| 3.13 through current release                | Azure CNI with {{prodname}} network policy                   |
```

File: calico-enterprise/getting-started/compatibility.mdx [hunk 3]
```diff
@@ -178,6 +148,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ------------------------------------ | ------------------- |
+| 3.17                 | {{prodname}} CNI with network policy | 1.24 - 1.25         |
```
Comment (sujeet-kr) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
| 3.17                 | {{prodname}} CNI with network policy | 1.25 - 1.26         |
```

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 4]
```diff
@@ -37,79 +37,46 @@ Note that all Windows feature limitations are described in [Windows limitations]
...
-| 3.10                 | Azure CNI with {{prodname}} network policy                                               |
-| 3.9                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.8                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.7                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.6                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.5                  | Azure CNI with {{prodname}} network policy                                               |
+| {{prodname}} version | {{prodname}} support                                         |
+| ---------------------------- | ------------------------------------------------------------ |
+| 3.14 through current release | - {{prodname}} CNI with network policy<br />- Azure CNI with {{prodname}} network policy |
+| 3.13 to 3.25                 | Azure CNI with {{prodname}} network policy                   |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx]:
```suggestion
| 3.13 through current release                | Azure CNI with {{prodname}} network policy                   |
```

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx [hunk 5]
```diff
@@ -178,6 +148,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ------------------------------------ | ------------------- |
+| 3.17                 | {{prodname}} CNI with network policy | 1.24 - 1.25         |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx]:
```suggestion
| 3.17                 | {{prodname}} CNI with network policy | 1.25 - 1.26         |
```

File: calico-enterprise/getting-started/compatibility.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed

File: calico-enterprise/getting-started/compatibility.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx (hunk 5 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx]: Fixed

**Comment by sabags** [comment]
LGTM

**Pull request merged**